### PR TITLE
Add a bio file for samshiah

### PR DESCRIPTION
**Pull request by samshiah**
I'm really talkative and want to tell you all about this commit

BLAH BLAH BLAH

### CODE REVIEW

File: bios/samshiah-bio.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# About samshiah
+
+- Lives in San Francisco, CA
+- Avid Bay Area sports fan (Warriors, 49ers, Giants)
+- Fluent in two languages (Mandarin and English)
```
Comment (githubteacher) [inline bios/samshiah-bio.md:5]: That's amazing!

File: bios/samshiah-bio.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Crazy about dogs
```
Comment (bobamoscms) [inline bios/samshiah-bio.md:6]: You'd love our Golden Retriever puppy Natty Boh

File: bios/samshiah-bio.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# About samshiah
+
+- Lives in San Francisco, CA
+- Avid Bay Area sports fan (Warriors, 49ers, Giants)
```
Comment (samshiah) [inline bios/samshiah-bio.md]: Me too!

File: bios/samshiah-bio.md (hunk 2 above)
Comment (molly-holly) [inline bios/samshiah-bio.md:6]: Hi. What's your favorite dog breed?

Comment (samshiah) [inline bios/samshiah-bio.md:6]: @molly-holly I love them all! 

@bobamoscms I just dogsat a mini golden this past weekend, super cute!

**Comment by molly-holly** [comment]
:+1:  looks good!

**Pull request merged**